cpp/modmesh/universe/pymod/wrap_World.cpp

```diff
@@ -86,37 +87,14 @@ WrapPoint3d<T>::WrapPoint3d(pybind11::module & mod, const char * pyname, const c
         //
         ;
 
+    // Wrap for operators.
     (*this)
-        .def(
-            "__iadd__",
-            [](wrapped_type & self, wrapped_type const & o) -> auto &
-            {
-                self += o;
-                return self;
-            })
-        .def(
-            "__iadd__",
-            [](wrapped_type & self, value_type v) -> auto &
-            {
-                self += v;
-                return self;
-            })
-        .def(
-            "__isub__",
-            [](wrapped_type & self, wrapped_type const & o) -> auto &
-            {
-                self -= o;
-                return self;
-            })
-        .def(
-            "__isub__",
-            [](wrapped_type & self, value_type v) -> auto &
-            {
-                self -= v;
-                return self;
-            })
-        .def("__imul__", &wrapped_type::operator*=)
-        .def("__itruediv__", &wrapped_type::operator/=)
+        .def(py::self += py::self)
```

Comment:
See https://pybind11.readthedocs.io/en/stable/advanced/classes.html#operator-overloading .  I forgot to use the convenient helpers.